arviz/plots/jointplot.py

```diff
@@ -45,17 +42,22 @@ def jointplot(trace, varnames=None, figsize=None, textsize=None, kind='scatter',
     ax_hist_x : matplotlib axes, x (top) distribution
     ax_hist_y : matplotlib axes, y (right) distribution
     """
-    trace = trace_to_dataframe(trace[skip_first:], combined=True)
-    varnames = get_varnames(trace, varnames)
+
+    data = convert_to_xarray(data)
+    data = data.where(data.draw >= skip_first).dropna('draw')
```

Comment:
We should remove `skip_first` from every plot.

Reply:
can do!